### ISSUE TITLE
Unable to build locally: Error during build

### ISSUE THREAD

**Issue by michael-rivera-florencehc-com** [Netflix/conductor#3012]
**Describe the bug**
Using the conductor documentation, and trying to run this in a docker container.
I have an M1 processor. 
When going through the documented steps, https://conductor.netflix.com/gettingstarted/docker.html, and running docker-compose up, I get the following error:
ERROR: no matching manifest for linux/arm64/v8 in the manifest list entries

I then figured, I had to make changes for the M1, So I followed the specialized steps for M1, https://conductor.netflix.com/gettingstarted/source.html. However, when I run docker-compose build, the command fails with the following:

Execution failed for task ':conductor-grpc:generateProto'.
#11 96.88 > protoc: stdout: . stderr: /root/.gradle/caches/modules-2/files-2.1/com.google.protobuf/protoc/3.13.0/4133adcea5ad74f787528cd64bcdc49fc98f608b/protoc-3.13.0-osx-x86_64.exe: 5: Syntax error: Unterminated quoted string

I then proceeded to try an run locally from source. I have Java SE Development Kit 18.0.1.1 , installed. I run the step "../gradlew bootRun" and get the error "> Could not find tools.jar. Please check that /Library/Internet Plug-Ins/JavaAppletPlugin.plugin/Contents/Home contains a valid JDK installation." - I have verified the $JAVA_HOME path and it is correct. Since i have a valid installation, I am not sure what the problem is. 

**Details**
Conductor version: Latest
Persistence implementation: defaults
Queue implementation: defaults
Lock: defaults
Workflow definition:
Task definition:
Event handler definition:

**To Reproduce**
Steps to reproduce the behavior:Use the documentation for docker on an M1

**Expected behavior**
A running environment
**Screenshots**

**Additional context**

**Comment by manishkushwaha1412** [comment]
Even I am facing the same issue, cloned latest conductor code and while build its giving me error.

**Comment by jxu-nflx** [comment]
It is an known issue with M1: https://github.com/Netflix/conductor/issues/2975

**Comment by michael-rivera-florencehc-com** [comment]
Reviewing recommendations. I did read through the M1 issues but to no avail. However, I will try a fresh pull and rebuild.

**Comment by michael-rivera-florencehc-com** [comment]
I pulled down the latest version and ran through the steps
Instead of using version 18 of the JDK which the documentation suggest it supports I rolled my JDK to version 11. 
After I got version 11 installed and setup.
And based on the instructions and changes for M1
I was able to get the Server running

../gradlew bootRun

I then proceeded to the UI steps and got this running as well. So it appears to be a problem with version 18 of the JDK that may be causing this issue.

The instructions should be updated to reflect this:
![image](https://user-images.githubusercontent.com/94843926/180481406-af3e9e0e-4c8d-441c-a9e6-68de227fbf76.png)

Since my goal is to get the server and UI working and I was able to do it. I will close this ticket. However, I suggest either finding a way to support higher level JDK's or update the documentation
